mlflow/genai/utils/trace_utils.py

```diff
@@ -80,6 +79,16 @@ def __exit__(self, exc_type, exc_value, traceback):
         NoOpTracer.start_span = self.original
 
 
+def is_none_or_nan(value: Any) -> bool:
```

Comment:
q: why can't we reuse the one in `mlflow.genai.evaluation.utils`?

Reply:
This creates a circular import that is definitely fixable, but involves some significant refactoring that I feel will be fragile to maintain and will create headaches for the addition of further APIs in the genai namespace. Porting this util function over, while not DRY, is definitely the lesser of two evils in my opinion.